docs/deprecated.md

```diff
@@ -0,0 +1,3 @@
+⚠️ cozy-client-js is a depreacted librairy. We recommend you to use our new client : [Cozy Client](https://docs.cozy.io/en/cozy-client/getting-started/)
```

Comment:
```suggestion
⚠️ cozy-client-js is a deprecated library. We recommend to use our new client : [Cozy Client](https://docs.cozy.io/en/cozy-client/getting-started/)
```

Reply:
[Nitpick] No space before `:` in english